src/Fabulous.Core/ViewHelpers.fs

```diff
@@ -7,9 +7,6 @@ open System.Collections.Generic
 open System.Threading
 open Xamarin.Forms
 
-[<System.Obsolete("Please change 'Xaml.XYZ' to 'View.XYZ'", error=false)>]
```

Comment:
Removed the obsolete `Xaml.XYZ` because it's been a long time since we moved to `View.XYZ`.
Also it helped remove the dependency on Xamarin.Forms.Core.fs